### PR TITLE
Polishing for beta

### PR DESCRIPTION
**Pull request by OisinKyne**
## Summary

Fixing things like CDVC references

### CODE REVIEW

File: docs/int/quickstart/quickstart-mainnet.md [hunk 1]
```diff
@@ -5,21 +5,24 @@ description: Run a cluster on mainnet
 
 # Run a DV on mainnet
```
Comment (thomasheremans) [inline docs/int/quickstart/quickstart-mainnet.md:6]: Not sure about the purpose of this page. Is it supposed to be a full guide to a DV config on mainnet? If so, should we add the information about DKG/create cluster. Currently looks like a guide but wouldn't be practical to follow. Alternatively we can use it as a page displaying the most important steps and configs to do on mainnet vs the other guides.

File: docs/int/quickstart/update.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ To do this, follow these steps:
...
+  <TabItem value="group" label="charon-distributed-validator-node" default>
     <pre>
       <code>
         cd charon-distributed-validator-node
       </code>
     </pre>
   </TabItem>
   
-  <TabItem value="solo" label="Solo cluster">
+  <TabItem value="solo" label="charon-distributed-validator-cluster">
```
Comment (thomasheremans) [inline docs/int/quickstart/update.md:25]: Should we add info on a clean shutdown before git pull?

**Review by thomasheremans — APPROVED** [review]

**Review by xenowits — APPROVED** [review]

File: docs/int/quickstart/update.md (hunk 2 above)
Comment (OisinKyne) [reply docs/int/quickstart/update.md:25]: Do you need to shut down the containers before pulling an update? I would have thought, calling `docker compose up -d` again would have just detected what's different and made the necessary changes, no shutdown of e.g. the EL+CL if not required (which risks unclean shutdowns in itself)

File: docs/int/quickstart/quickstart-mainnet.md (hunk 1 above)
Comment (OisinKyne) [reply docs/int/quickstart/quickstart-mainnet.md:6]: Yeah we probably should just have one set of guides tabulated for the different networks, future problem though.

**Pull request merged**